Library/Formula/libswiften.rb

```diff
@@ -1,24 +1,13 @@
 class Libswiften < Formula
   desc "C++ library for implementing XMPP applications"
   homepage "https://swift.im/swiften"
-  revision 2
 
   stable do
-    url "https://swift.im/downloads/releases/swift-2.0/swift-2.0.tar.gz"
-    sha256 "cbcdbe527dc4d112a38f3cdec5f1051d3beba0b97f8082f90debd04a5b45a41f"
+    url "git://swift.im/swift", :tag => "swift-3.0", :revision => "14ddf8b470f5a3420b5f2c96daea33c2513cac6e"
```

Comment:
Ok, this tarball works for me, too 😊
The patch is still needed, though. Otherwise, the build fails with:

```bash
==> /usr/local/opt/scons/bin/scons -j4 V=1 optimize=1 debug=0 allow_warnings=1 swiften_dll=1 boost_includedir=/usr/local/Cellar/boost/1.60.0_1/include boost_libdir=/usr/local/Cellar/boost/1.60.0_1/lib libidn_includedir=/usr/local/Cellar/libidn/1.32/include libidn_libdir=/usr/local/Cellar/libidn/1.32/lib SWIFTEN_INSTALLDIR=/usr/local/Cellar/libswiften/3.0 openssl=no SLUIFT_INSTALLDIR=/usr/local/Cellar/libswiften/3.0 lua_includedir=/usr/local/Cellar/lua/5.2.4_3/include lua_libdir=/usr/local/Cellar/lua/5.2.4_3/lib /usr/local/Cellar/libswiften/3.0

Last 15 lines from /Users/manfred/Library/Logs/Homebrew/libswiften/01.scons:
Install file: "Swiften/Crypto/OpenSSLCryptoProvider.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Crypto/OpenSSLCryptoProvider.h"
Install file: "Swiften/Crypto/PlatformCryptoProvider.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Crypto/PlatformCryptoProvider.h"
Install file: "Swiften/Crypto/WindowsCryptoProvider.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Crypto/WindowsCryptoProvider.h"
Install file: "Swiften/Disco/CapsInfoGenerator.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/CapsInfoGenerator.h"
Install file: "Swiften/Disco/CapsManager.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/CapsManager.h"
Install file: "Swiften/Disco/CapsMemoryStorage.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/CapsMemoryStorage.h"
Install file: "Swiften/Disco/CapsProvider.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/CapsProvider.h"
Install file: "Swiften/Disco/CapsStorage.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/CapsStorage.h"
Install file: "Swiften/Disco/ClientDiscoManager.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/ClientDiscoManager.h"
Install file: "Swiften/Disco/DiscoInfoResponder.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/DiscoInfoResponder.h"
Install file: "Swiften/Disco/DiscoServiceWalker.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/DiscoServiceWalker.h"
Install file: "Swiften/Disco/DummyEntityCapsProvider.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/DummyEntityCapsProvider.h"
Install file: "Swiften/Disco/EntityCapsManager.h" as "/usr/local/Cellar/libswiften/3.0/include/Swiften/Disco/EntityCapsManager.h"
scons: *** [Swiften/libSwiften.3.0.dylib] Source file: Swiften/TLS/SecureTransport/SecureTransportContext.o is static and is not compatible with shared target: Swiften/libSwiften.3.0.dylib
scons: building terminated because of errors.
```

Updated this PR accordingly.